### PR TITLE
[WIP][13.0] account_financial_report: migrate

### PR DESCRIPTION
**Pull request by i-vyshnevska**
waiting for migration of date_range, report_xlsx

### CODE REVIEW

File: account_financial_report/models/account_move_line.py [hunk 1]
```diff
@@ -0,0 +1,30 @@
+# Copyright 2019 ACSONE SA/NV (<http://acsone.eu>)
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).-
+from odoo import api, models
```
Comment (ernestotejeda) [inline account_financial_report/models/account_move_line.py:3]: 'odoo.api' imported but unused
https://travis-ci.org/OCA/account-financial-reporting/jobs/592101257#L339

**Review by ernestotejeda — CHANGES_REQUESTED** [review]
Add a file called oca_dependencies.txt in the repository root and put server-ux inside it so that the date_range module can be found.

You can add this too:
reporting-engine https://github.com/schout-it/reporting-engine 13.0-mig-report_xlsx
until report_xlsx is mixed and so this module can be tested

It will finally look like this:
server-ux
reporting-engine https://github.com/schout-it/reporting-engine 13.0-mig-report_xlsx

**Comment by patel1008b** [comment]
@i-vyshnevska Thanks for migration, its ready to use?

**Comment by i-vyshnevska** [comment]
this pr is blocked by other modules migration and there already new commits it should be redone

**Comment by patel1008b** [comment]
@i-vyshnevska @ernestotejeda 

https://github.com/OCA/reporting-engine/pull/331
https://github.com/OCA/server-ux/pull/95
https://github.com/OCA/server-ux/tree/13.0/date_range

It would be nice to have this PR merge. It's a very useful module.

**Comment by patel1008b** [comment]
@i-vyshnevska @ernestotejeda

**Comment by patel1008b** [comment]
@i-vyshnevska @ernestotejeda any further updates for this PR. As it's main for accounting application to have these reports.

**Pull request closed without merging**